### PR TITLE
Created support for IP-cameras through RSTP

### PR DESCRIPTION
**Pull request by durgabhavaniv**
Able to take take input form ip cameras for running demo samples for multichannel face detection and human pose estimation

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/common/samples/args_helper.hpp [hunk 1]
```diff
@@ -31,6 +31,7 @@
 * @param arg path to a file to be checked for existence
 * @return files updated vector of verified input files
 */
+//void readInputFilesArguments(std::vector<std::string> &files, const std::string& arg) {
```
Comment (vladimir-dudnik) [inline demos/common/samples/args_helper.hpp]: is there are reason to keep commented out lines of code? Please remove those.

File: demos/common/samples/args_helper.hpp [hunk 2]
```diff
@@ -73,6 +74,7 @@ inline void readInputFilesArguments(std::vector<std::string> &files, const std::
 *        It's necessary to process multiple values for single key
 * @return files updated vector of verified input files
 */
+// void parseInputFilesArguments(std::vector<std::string> &files) {
```
Comment (vladimir-dudnik) [inline demos/common/samples/args_helper.hpp]: do not keep commented out lines of code

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by durgabhavaniv** [comment]
@vladimir-dudnik can i view details wht build was not successful

**Comment by vladimir-dudnik** [comment]
@durgabhavaniv the issue seems on our infrastructure side. We are working to fix that.

12:34:42  Test case #11: -d_act CPU -d_fd CPU -d_lm CPU -d_reid CPU -i /tmp/tmpm2k03ngr/image_seq/smart-classroom-demo/input-%04d.JPEG -m_act /tmp/tmpm2k03ngr/models/intel/person-detection-raisinghand-recognition-0001/FP32/person-detection-raisinghand-recognition-0001.xml -m_fd /tmp/tmpm2k03ngr/models/intel/face-detection-adas-0001/FP32/face-detection-adas-0001.xml -m_lm /tmp/tmpm2k03ngr/models/intel/landmarks-regression-retail-0009/FP32/landmarks-regression-retail-0009.xml -no_show
12:34:42  
12:34:43  [ INFO ] InferenceEngine: 
12:34:43  	API version ............ 2.1
12:34:43  	Build .................. custom_releases/2019/R3_cb6cad9663aea3d282e0e8b3e0bf359df665d5d0
12:34:43  	Description ....... API
12:34:43  [ INFO ] Parsing input parameters
12:34:43  [ INFO ] Reading video '/tmp/tmpm2k03ngr/image_seq/smart-classroom-demo/input-%04d.JPEG'
12:34:43  [ INFO ] Loading Inference Engine
12:34:43  [ INFO ] Device info: 
12:34:43  	CPU
12:34:43  	MKLDNNPlugin version ......... 2.1
12:34:43  	Build ........... 30677
12:34:43  
12:34:43  [ WARNING ] Face recognition models are disabled!
12:34:43  To close the application, press 'CTRL+C' here
12:34:43  [ ERROR ] action_sum_exp_values can't be equal to 0
12:34:43  
12:34:43  Exit code: 1

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request closed without merging**